cds_ils/importer/importer.py

```diff
@@ -148,13 +148,26 @@ def index_all_records(self):
         series_indexer = current_app_ils.series_indexer
         eitem_indexer = current_app_ils.eitem_indexer
 
-        eitem = self.eitem_importer.eitem_record
-        if eitem:
+        document_class = current_app_ils.document_record_cls
+        series_class = current_app_ils.series_record_cls
+        eitem_class = current_app_ils.eitem_record_cls
+
+        eitem_imported = self.eitem_importer.eitem_record
+        if eitem_imported:
+            # refetch record
+            eitem = eitem_class.get_record_by_pid(eitem_imported["pid"])
```

Comment:
Why do we need to refetch the record every time?

Reply:
to be sure we get the latest version, since the import is performed in the celery task